### ISSUE TITLE
WW exercises are showing solutions in HTML

### ISSUE THREAD

**Issue by Alex-Jordan** [rbeezer/mathbook#957]
Following the work extending from #953, the HTML output of a WW exercise is showing the solution, even with the stringparam set to no. At least true for an exercisegroup exercise, possibly true for all divisional exercises.

In one example in ORCCA's merge file, the structure where this happens is
```
<exercises>
  <exercisegroup cols="3">
    <exercise>
      <webwork-reps xml:id="extracted-webwork-18" ww-id="webwork-18">
        <pg source="BasicAlgebra/SignedNumbersArithemtic/AdditionWithNegativeNumbers10.pg"/>
        <static source="BasicAlgebra/SignedNumbersArithemtic/AdditionWithNegativeNumbers10.pg" seed="18">
          <statement><p>....
          ...</p></statement>
      
          <solution><p>...</p></solution>
        </static>
      
        <server-url hint="yes" solution="yes">...</server-url>
        <server-url hint="yes" solution="no">...</server-url>
        <server-url hint="no" solution="yes">...</server-url>
        <server-url hint="no" solution="no">...</server-url>
      </webwork-reps>
    </exercise>
    ...
```

**Comment by Alex-Jordan** [comment]
OK, I think I understand. The webwork tests are still using the old `exercise.text.solution` stringparam and similar. So this would have been an issue before as well. But the last time I built ORCCA, I had both the old and the new stringparams set for good measure. In the interim, I cut the old kind from my build.

So it is *not* a result of the recent fix. It's an entirely separate issue.

**Comment by rbeezer** [comment]
I think it was intentional neglect when I did all this work.

Does this mean that using the old switches is a reasonable workaround for right now?

**Comment by rbeezer** [comment]
I think this has been addressed at, or near, 3cbd75031d338cc6fefc6112b901086f342395f9  Re-open if I am mistaken.

**Comment by rbeezer** [comment]
But also see #965
